R/find_missing_ingredients.R

```diff
@@ -1,17 +1,26 @@
 #' find_missing_ingredients Documentation
 #'
-#' For each column with missing values, this function will create a reference
-#' list of row indices, sum the number and calculate proportion of missing values
+#' Example Description: Returns the indices of rows with columns which have missing values,
+#' as well as how many missing values they have
 #'
-#' @param df A dataframe
+#' @param data
 #'
```

Comment:
```suggestion
#' @param data A data frame that want to check missing values
#'
```

Add explanation to the parameter.